### ISSUE TITLE
Quitting on Serverless 1.33

### ISSUE THREAD

**Issue by pianomansam** [aheissenberger/serverless-appsync-offline#8]
When running `serverless appsync-offline start`, ever since upgrading to Serverless 1.33, the command quits as soon as the GraphQL server comes up. From my logs, when using nodemon:

```
ℹ️  appsync-emulator:serverCore server bound { address: '::', family: 'IPv6', port: 62222 }
Serverless: AppSync started: http://localhost:62222/graphql
[nodemon] clean exit - waiting for changes before restart
```

You'll notice that while it exits cleanly, it doesn't do any of the cleanup. Otherwise we would see a line like `AppSync offline - stopping graphql and dynamoDB`.

I'm not sure if the bug is in this plugin or appsync-emulator-serverless.

**Comment by aheissenberger** [comment]
@pianomansam have a try with the updated AppSync Emulator

**Comment by pianomansam** [comment]
Using Serverless 1.34 doesn't cause this issue.

**Comment by aheissenberger** [comment]
@pianomansam  thanks for updating this issue with this positive news :-)

**Comment by mattgibson** [comment]
Serverless 1.34.0 does cause this issue for me as well as the latest version, 1.37.1. If I downgrade to 1.32.0, then it's OK.

**Comment by mattgibson** [comment]
This seems to be the cause of the issue:

https://github.com/serverless/serverless/pull/5349/commits/a0a0fb1530fa17e9f215879dfdead88f61e46169

**Comment by jaydp17** [comment]
@aheissenberger this issue does occur on the current master.
I tried out @mattgibson's fix and it seems to work!!

Maybe we can merge the PR?

**Comment by aheissenberger** [comment]
> @aheissenberger this issue does occur on the current master.
> I tried out @mattgibson's fix and it seems to work!!
> 
> Maybe we can merge the PR?

this is a fix to the serverless/serverless repository - you need to ask the serverless team to merge the fix

**Comment by jaydp17** [comment]
@aheissenberger sorry, I didn't see that, thanks for informing.

**Comment by aheissenberger** [comment]
is fixed with version 1.3.2
